### DISCUSSION TITLE
How to update sniper replit

### DISCUSSION THREAD

**Discussion by Jeshua09090** [marioparaschiv/nitro-sniper#267] [Q&A] [answered]
Hi ¿is there a command to update nitro sniper in replit?

**Comment by MrD4rio** [comment, marked as answer]
Check https://github.com/slow/nitro-sniper/wiki/Installation
Follow the Installation instruction for repl.it, if you do everything correctly you will have the Sniper auto-updating.

**Reply by Jeshua09090** [reply]
Thank you!
